doc/conf.py

```diff
@@ -525,6 +525,7 @@
     'current_version': version,
     'versions': (("latest", "/"),
                  ("2.0.0", "/2.0.0/"),
+                 ("1.14.1", "/1.14.1/"),
                  ("1.14.0", "/1.14.0/"),
```

Comment:
can we remove 1.14.0?

Reply:
We could, but  since that's the original LTS release, I figured folks might want to be able to get to that documentation as well as the 1.14.1 (will everyone move from 1.14.0 to 1.14.1?)

Reply:
@nashif Are you waiting for me to remove the 1.14.0 link?... let me know.

Reply:
I'm not sure what our general thinking is on links to 1.14.0 and every other potential LTS release. I'm okay with either choice.